career-growth.md

```diff
@@ -0,0 +1,69 @@
+
+> The above question is vague, purposefully so. You may include some, all, or none of the following: Are there any big skills you want to get that you just don't have now? Do you have a desired job title? Desired responsibilities? 
+
+### What are the biggest gaps between your current skills, vision and expertise and those you would need to achieve the goal above?
+
+> One useful way to think about this, if you were submitting your resume to apply for your dream, what issues might a hiring manager flag up?
+
+### How will you close these gaps? How does closing these gaps help you to deliver on Packback's mission?
+
+> In the spirit of the alliance between you and Packback, let's think through the other half of this. When your expertise, vision and skills grow, how will your role at Packback transform and push Packback even faster towards its mission?
+
+> Here we can attach some more concrete goals, on a timescale of 1-2 years, that you want to take to your manager. What is your mission for the next year or two? How will that help you? How will it help Packback?
+
+## Conscious Career Growth Guide for Managers (of Managers)
+
+A quick reminder about the intention of this exercise, to be summarized at the start of all conversations:
+
+* This is not a performance conversation; this is directional and aspirational.
+* There is no failing here.
+* Winning means leaving the meeting excited about future growth, how that growth helps the employee's growth, and how in turn the employee's growth helps Packback.
+
+### Running Meeting 1 (Managers of Managers)
+
+You may not be in this employee's day-to-day life, but you likely are in the sort of role they see themselves in a few years, or, even if you are not, you definitely have significantly more experience and can give concrete guidance around their growth.
+
+Success for you in this meeting is threefold:
+
+* To give the employee the tools to have really productive career conversations with their direct manager.
+* To leave the meeting with a shared excitement with the employee about their future.
+* For the employee to be able to make a direct link between their own growth towards their ideal role and how their growth accelerates Packback.
+
+If you're running these meetings you have been managing people for a while, so there's not a prescribed formula for it. Just be you! A manager of managers at Packback is a strong people person, and you know what you're doing.
+
+### Running Meeting 2 (Managers)
+
+During this meeting, the employee should come prepared with answers to the three questions. You get to be the one who dives into the nitty-gritty of how they're going to do these things.
+
+You should read the answers ahead of time, **if there is a big disconnect between the employee's ideal career direction and their current role, escalate quickly and decisively to your manager, who ran the first meeting with them.** We can work together to figure out a path forward.
```

Comment:
The way this is wording makes me feel like it's a bit contradictory to the whole "no wrong answers" thing. I would appreciate at least 1-2 details or an example about what figuring out a path forward means in this situation.

Reply:
Honestly, I have a hard time imagining a case where this would happen. This would be a situation where, say, an engineer says their long-term career goal is to be a CFO. Our ability to support them in learning the skills they would need for that are be pretty limited. If that really is their long term goal, we could explore finding them another position at Packback that suits them, or perhaps Packback isn't a long term fit for them. Being something completely different isn't a wrong answer, but we're definitely not going to be able to provide them with the opportunities they need.